packages/hammer/src/setup.ts

```diff
@@ -1,26 +1,30 @@
 import { Managers, Transactions } from "@arkecosystem/crypto";
+import { ProtokolConnection } from "@protokol/client";
 import { Transactions as NFTTransactions } from "@protokol/nft-base-crypto";
 import { Transactions as NFTExchangeTransactions } from "@protokol/nft-exchange-crypto";
 
 import { configurations } from "./configurations";
-import exceptions from "./data/networks/exceptions.json";
-import milestones from "./data/networks/milestones.json";
-import configs from "./data/networks/network.json";
 
 export const setupScript = async () => {
-	Managers.configManager.setFromPreset(configurations.network as "testnet" | "devnet" | "mainnet");
-	Managers.configManager.setHeight(configurations.networkHeight);
+	const client = new ProtokolConnection(configurations.clientHost);
+	const configs = await client.api("node").crypto();
 
 	Managers.configManager.setConfig({
```

Comment:
You can use the whole crypto config that you get from the node.
```suggestion
	Managers.configManager.setConfig(configs);
```

Reply:
You can't, because genesis block parameters don't fit in.
I could ts-ignore it or something but anyways genesis block doesn't impact crypto create transactions.